src/index.ts

```diff
@@ -33,56 +38,81 @@ export default class HuddlyDeviceAPIUSB implements IHuddlyDeviceAPI {
     return this.deviceDiscoveryManager;
   }
 
-  async getValidatedTransport(device): Promise<ITransport> {
-    if ([HuddlyHEX.GO_PID, HuddlyHEX.L1_PID, HuddlyHEX.BASE_PID].includes(device.productId)) {
+  async getValidatedTransport(device: usb.Device): Promise<ITransport> {
+    if (
+      [HuddlyHEX.GO_PID, HuddlyHEX.L1_PID, HuddlyHEX.BASE_PID].includes(
+        (device as any as UsbDevice).productId
```

Comment:
Is this casting to any before casting to UsbDevice intentional?

Reply:
Yes, because typescript will complain that product id is not present in a usb.Device type. But i know for sure that the incoming device argument will contain the information described in the UsbDevice class. So yes, it is intentional.